WordPress/src/main/res/layout/stats_widget_title_dark.xml

```diff
@@ -8,6 +8,8 @@
     android:layout_height="@dimen/stats_widget_list_item_height"
     android:background="@drawable/bg_rectangle_gray_700_top_radius_4dp"
     android:orientation="horizontal"
+    android:visibility="gone"
+    tools:visibility="visible"
```

Comment:
I wonder why we make visibility gone here, then make it visible programmatically. What is the reason? 🤔

Reply:
It is for the widget previews.  The title is separate included layout along with ListView in that layout at the top.  Title is not an item row like the data.  Previews need their titles separately!